### DISCUSSION TITLE
explode(): Passing null to parameter #2

### DISCUSSION THREAD

**Discussion by mrahmadt** [spatie/nova-tags-field#168] [Q&A]
Hello 

I followed the readme installation steps and added the Tag field to my resource. However, I keep getting this wanting in my log and no Tag is created.

Not sure if I need something extra with Nova 4

```
[2022-09-10 08:17:20] laravel.WARNING: explode(): Passing null to parameter #2 ($string) of type string is deprecated in /var/www/html/vendor/spatie/nova-tags-field/src/Tags.php on line 112  
```

**Comment by freekmurze** [comment]
In our projects, it seems to work fine. I'd accept a PR that fixes this for you.

**Reply by Gavrisimo** [reply]
@freekmurze yeah it works fine, it's just that there's a lot of noise in logs because of this.

I have created a PR that should help when dealing with `null` values: https://github.com/spatie/nova-tags-field/pull/179
